com.civicrm.hrjobroles/CRM/Hrjobroles/BAO/HrJobRoles.php

```diff
@@ -78,6 +79,11 @@ public static function getDepartmentsList($contractID)  {
               INNER JOIN civicrm_option_group cog
                 ON cov.option_group_id = cog.id
               WHERE hrjr.job_contract_id = %1
+                AND hrjr.start_date <= '$today'
```

Comment:
it is a common thing we do here  is to put variables inside strings inside curely braces , so it should be something like :

AND hrjr.start_date <= '{$today}'